templates/Pages/home.php

```diff
@@ -147,7 +147,7 @@
 
                         <?php $settings = Cache::getConfig('_cake_core_'); ?>
                         <?php if (!empty($settings)) : ?>
-                            <li class="bullet success">The <em><?= $settings['className'] ?>Engine</em> is being used for core caching. To change the config edit config/app.php</li>
+                            <li class="bullet success">The <em><?= $settings['className'] ?></em> is being used for core caching. To change the config edit config/app.php</li>
```

Comment:
h() would also be good to add

Reply:
`$settings['className']` comes from config file , not user input

Reply:
Still good practice imo

Reply:
I can add it separately. We should have it if only to model safe template authoring.